admin/settings.yaml

```diff
@@ -756,6 +756,11 @@ hs-toxcore:
           - "build / stack"
           - "checks / check-release"
           - "common / buildifier"
+          # Netlify tests:
+          - "Header rules - hs-toxcore"
+          - "Pages changed - hs-toxcore"
+          - "Redirect rules - hs-toxcore"
+          - "netlify/hs-toxcore/deploy-preview"
```

Comment:
Where do you get the names of the checks from? I don't see those being listed in https://github.com/TokTok/hs-github-tools/pull/180/checks nor being defined in `TokTok/hs-github-tools/blob/master/.github/workflows/*.yml` or `TokTok/ci-tools/blob/master/.github/workflows/*.yml`.

Reply:
Oh, it's `hs-toxcore`, not `hs-github-tools`. Let me see if I can find them in there...

Reply:
Ok, I see. In our CI setup everything one repo runs is defined in some other repo, or two, or three, or none as it's done manually via API request (like this `admin/settings.yml`), so it can be rather confusing to unravel where are things set or defined. I get that it's done to reduce the CI rule duplication and the need to create 30 identical PRs, one for every TokTok repo -- just need to wrap my head around of how it's all setup.